src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.hpp

```diff
@@ -34,13 +35,27 @@ class er;
 namespace VariableFixing {
 
 /// \ingroup VariableFixingGroup
-/// \brief Fix conservative variables
+/// \brief Fix conservative variables using method developed by Foucart.
```

Comment:
I'm not sure why, it might be something to do my rendering script, but I can't find the generated documentation. @wthrowe could you check this please? :)

Reply:
There are a bunch of doxygen warnings, that's probably the issue.
```
/home/wthrowe/tmp/host-docs/source/src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.hpp:51: warning: End of comment block while inside formula.
/home/wthrowe/tmp/host-docs/source/src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.hpp:45: warning: End of comment block while inside formula.

/home/wthrowe/tmp/host-docs/source/src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.cpp:29: warning: documented symbol `VariableFixing::FixConservatives::FixConservatives' was not declared or defined.
/home/wthrowe/tmp/host-docs/source/src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.cpp:41: warning: no uniquely matching class member found for 
  void VariableFixing::FixConservatives::pup(PUP::er &p)

/home/wthrowe/tmp/host-docs/source/src/Evolution/Systems/GrMhd/ValenciaDivClean/FixConservatives.cpp:56: warning: no uniquely matching class member found for 
  void VariableFixing::FixConservatives::operator()(const gsl::not_null< Scalar< DataVector > * > tilde_d, const gsl::not_null< Scalar< DataVector > * > tilde_tau, const gsl::not_null< tnsr::i< DataVector, 3, Frame::Inertial > * > tilde_s, const tnsr::I< DataVector, 3, Frame::Inertial > &tilde_b, const tnsr::ii< DataVector, 3, Frame::Inertial > &spatial_metric, const tnsr::II< DataVector, 3, Frame::Inertial > &inv_spatial_metric, const Scalar< DataVector > &sqrt_det_spatial_metric) const
```